pandas/tests/groupby/test_function.py

```diff
@@ -1636,3 +1636,20 @@ def test_apply_to_nullable_integer_returns_float(values, function):
     result = groups.agg([function])
     expected.columns = MultiIndex.from_tuples([("b", function)])
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_sum_below_mincount_nullable_integer():
+    # https://github.com/pandas-dev/pandas/issues/32861
+    df = pd.DataFrame({"a": [0, 1, 2], "b": [0, 1, 2], "c": [0, 1, 2]}, dtype="Int64")
+    grouped = df.groupby("a")
+    idx = pd.Index([0, 1, 2], dtype=object, name="a")
+
+    result = grouped["b"].sum(min_count=2)
+    expected = pd.Series([np.nan] * 3, index=idx, name="b")
+    tm.assert_series_equal(result, expected)
+
+    result = grouped.sum(min_count=2)
+    expected = pd.DataFrame(
+        {"b": [pd.NA] * 3, "c": [pd.NA] * 3}, dtype="Int64", index=idx
```

Comment:
Not ideal but it seems we get `NA` in the `DataFrame` case but `NaN` for `Series`. Should I add a FIXME comment with a follow-up issue?

Reply:
huh? that seems odd, can you track this down

Reply:
can you create an issue about this. I am not sure this is correct.

Reply:
Which part seems incorrect? The dtype of the index being object is maybe odd, but otherwise seems okay?

Reply:
it seems your comment about is not correct, e.g. we have NA in all cases. is that not what you meant here?

Reply:
I think that comment was from before adding the conversion using maybe_cast_result (so the original test had NaN for Series but NA for DataFrame): https://github.com/pandas-dev/pandas/pull/32914/commits/2a3f8145769d055886a48bc6b9c6dc4e7ed257e3

Reply:
ok this is fine then